hugo/content/citation/_index.md

```diff
@@ -0,0 +1,91 @@
+It aims to promote the identification, discovery, and reuse of research resources. Research Resource Identifiers (**RRIDs**) are persistent and unique identifiers for
+referencing a research resource. For official guidance see the [SciCrunch page on RRID citations](https://scicrunch.org/resources/about/guidelines).
+
+The RRID for the Swedish Reference Genome Portal is [**XXXXXXXX**](https://XXXXXXXXXX).
+
+By citing the portal using the RRID, you will facilitate further reuse of the portal, enable us to track that activity, and allow others to easily find the _Summary Report_
+for usage of the Swedish Reference Genome Portal.
+
+**In-text citation (APA format)**:\
+Swedish Reference Genome Portal (SciLifeLab Data Centre; _version number_; RRID: XXXXXXXXX).
+
+**Reference list (APA format)**:\
+Swedish Reference Genome Portal (_access date_), SciLifeLab Data Centre. Version (version number). <https://genomes.scilifelab.se>, RRID:XXXXXXXX. [Species DOI].
+
+You will find the version number of the portal at XXXXXthe bottom of the footer on any pageXXXXXX, or on our
+<a target="_blank" href="https://github.com/ScilifelabDataCentre/genome-portal/">genome-portal repository</a> under 'releases'.
+
+### 2. Citing a specific species entry
+
+If you would like to refer to a specific species entry from the Genome Portal, you can do so by taking the Genome Portal citation (Section 1) and adding the species name and the
+DOI pointing to its associated metadata in SciLifeLab Data Repository:
+
+**In-text citation (APA format)**:\
+The [_Species name_] entry in the Swedish Reference Genome Portal (SciLifeLab Data Centre; _version number_; RRID: XXXXXXXXX; [Species DOI]).
+
+**Reference list (APA format)**:\
+Swedish Reference Genome Portal, [_Species name_] entry (_access date_). SciLifeLab Data Centre. Version (version number). <https://genomes.scilifelab.se/[species_name>], RRID:XXXXXXXX. [DOI].
+
+where `[species_name]` should be replaced with the binomal name of your species in lower case letters and an underscore, and `[DOI]` with the DOI of the associated metadata in SciLifeLab Data Repository.
+
+#### Data availability statement for submitting researchers
+
+Researchers who have submitted their data to the Genome Portal may want to include the following text for the Data Availability statement in their research manuscript.
+
+"Visualisations of the genome assemby and annotation tracks from this publication can be found in the Swedish Reference Genome Portal at <https://genomes.scilifelab.se/[species_name>].
+The accompanying metadata for this entry in the portal can be found at [DOI]."
+
+where `[species_name]` and `[DOI]` should be replaced as described above.
+
+The Genome Portal staff can help you reserve an entry on the portal and a DOI for the metadata already during the manuscript submission stage. When the manuscript is accepted for publication and the data has been
```

Comment:
This part could possibly be moved to the "Contribute" section

Reply:
I like the idea. It would reduce the amount of text and hopefully make the citation page slightly more readable. I drafted a little something in [46b7e1f](https://github.com/ScilifelabDataCentre/genome-portal/pull/55/commits/46b7e1f4d7f94a6b570e5f52a868a6d3e3d5500e).